lib/bandit/http2/stream_collection.ex

```diff
@@ -35,15 +35,13 @@ defmodule Bandit.HTTP2.StreamCollection do
     delta = initial_send_window_size - collection.initial_send_window_size
 
     streams =
-      collection.streams
-      |> Enum.map(fn
+      Map.new(collection.streams, fn
         {id, %Stream{state: state} = stream} when state in [:open, :remote_closed] ->
           {id, %{stream | send_window_size: stream.send_window_size + delta}}
 
         {id, stream} ->
           {id, stream}
       end)
-      |> Map.new()
```

Comment:
This ends up as the exact same implementation internally (see https://github.com/elixir-lang/elixir/blob/a64d42f5d3cb6c32752af9d3312897e8cd5bb7ec/lib/elixir/lib/map.ex#L217).

I have a mild stylistic preference for the existing implementation (they're both fine! it's just a matter of convention in that I find explicit maps to be more legible, and it's also the convention used elsewhere in the project.

Reply:
Sure! I reverted this one.